scripts/delayed.js

```diff
@@ -1,7 +1,32 @@
 // eslint-disable-next-line import/no-cycle
-import { sampleRUM } from './aem.js';
+import { sampleRUM, loadScript } from './aem.js';
 
 // Core Web Vitals RUM collection
 sampleRUM('cwv');
 
-// add more delayed functionality here
+/*
+  * Returns the environment type based on the hostname.
+*/
+function getEnvType(hostname = window.location.hostname) {
+  const fqdnToEnvType = {
+    'commonmoves.com': 'preview',
```

Comment:
Why is this preview and not live?

Reply:
I have it set to preview out of an abundance of caution not to impact their analytics by accident.

Reply:
Added preview options and set customer domain to live.